### ISSUE TITLE
Option to disable site engagement statistics

### ISSUE THREAD

**Issue by alfredonodo** [brave/brave-browser#33791]
Hi,
since Brave is a a privacy oriented browser, it should give the possibility to disable [site engagement statistics](https://www.chromium.org/developers/design-documents/site-engagement/) that [can leak frequently visited sites](https://fingerprint.com/blog/potential-privacy-implications-lookalike-warnings-chromium/).
Thank you

**Comment by manikandancode** [comment]
When users delete the browsing history, engagement scores are cleared. 
Users may load brave://settings/clearBrowserData in the browser's address bar select "all time" in the time range menu and make sure that the browsing history is checked and click Clear Data.  Restart the web browser.

**Comment by alfredonodo** [comment]
> When users delete the browsing history, engagement scores are cleared. Users may load brave://settings/clearBrowserData in the browser's address bar select "all time" in the time range menu and make sure that the browsing history is checked and click Clear Data. Restart the web browser.

Right. However, Brave should have an option to disable it and above all it should be disabled by default.

**Comment by akovia** [comment]
I'd also like to add that this follows you into incognito mode as well. 

I thought Brave would at least circumvent this behavior when using incognito mode, and was surprised to find out it doesn't. Would it be possible to have a button to clear this data wholesale, or even a button beside the `Cookies and site data` when you `View site information` to clear that specific site from the list? 

I just find it disconcerting that incognito mode is not at all. 
This site dispels any notion of that.

https://fingerprint.com/
`Try opening [fingerprint.com](https://fingerprint.com/) in an Incognito or Private window. Your visitor ID will stay the same.`
